src/context.rs

```diff
@@ -1251,6 +1282,24 @@ impl<'context> WidgetContext<'context> {
         self.window
     }
 
+    /// Returns the locale for this widget.
+    #[must_use]
+    pub fn locale(&self) -> &Value<LanguageIdentifier> {
+        &self.locale
+    }
+
+    /// Returns the current translation bundle for this widget.
+    pub fn translation(
+        &self
+    ) -> &FluentBundle<FluentResource> {
+        if let Some(bundle) = self.translations.loaded_translations.get(&self.locale.get()) {
```

Comment:
`self.locale.get()` accesses the current locale, but this function isn't tracking the invalidation of the locale. I think `self.locale.get_tracking_invalidate(self)` should work here.